src/_includes/layouts/base.njk

```diff
@@ -37,11 +37,11 @@ eleventyComputed:
 
     <!-- Browser Title -->
     {% if navTitle %}
-    <title>{{ navTitle }} &#x2022; FlowFuse{% if page.url.match('\/handbook\/.+') %} Handbook{% endif %}{% if page.url.match('\/docs\/.+') %} Docs{% endif %}</title>
+    <title>{{ navTitle }} &#x2022; FlowFuse{% if page.url and page.url.match('\/handbook\/.+') %} Handbook{% endif %}{% if page.url and page.url.match('\/docs\/.+') %} Docs{% endif %}</title>
     {% elif meta and meta.title %}
-    <title>{{ meta.title }} &#x2022; FlowFuse{% if page.url.match('\/handbook\/.+') %} Handbook{% endif %}{% if page.url.match('\/docs\/.+') %} Docs{% endif %}</title>
+    <title>{{ meta.title }} &#x2022; FlowFuse{% if page.url and page.url.match('\/handbook\/.+') %} Handbook{% endif %}{% if page.url and page.url.match('\/docs\/.+') %} Docs{% endif %}</title>
     {% elif title %}
-    <title>{{ title }} &#x2022; FlowFuse{% if page.url.match('\/handbook\/.+') %} Handbook{% endif %}{% if page.url.match('\/docs\/.+') %} Docs{% endif %}</title>
+    <title>{{ title }} &#x2022; FlowFuse{% if page.url and page.url.match('\/handbook\/.+') %} Handbook{% endif %}{% if page.url and page.url.match('\/docs\/.+') %} Docs{% endif %}</title>
```

Comment:
`git checkout origin/main -- src/_includes/layouts/base.njk` is the command you'll likely need

Reply:
This was necessary, otherwise, if there's a post with future date I get this error:
```
[11ty] File changed: src/_includes/layouts/base.njk
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] 1. Having trouble writing to "false" from "./src/blog/2023/01/test.md" (via EleventyTemplateError)
[11ty] 2. (./src/_includes/layouts/base.njk) [Line 19, Column 65]
[11ty]   Error: Unable to call `page["url"]["match"]`, which is undefined or falsey (via Template render error)
[11ty] 
[11ty] Original error stack trace: Template render error: (./src/_includes/layouts/base.njk) [Line 19, Column 65]
[11ty]   Error: Unable to call `page["url"]["match"]`, which is undefined or falsey
[11ty]     at Object._prettifyError (/Users/yndira/Documents/Yndira/flowforge/website/node_modules/nunjucks/src/lib.js:32:11)
[11ty]     at /Users/yndira/Documents/Yndira/flowforge/website/node_modules/nunjucks/src/environment.js:464:19
[11ty]     at Template.root [as rootRenderFunc] (eval at _compile (/Users/yndira/Documents/Yndira/flowforge/website/node_modules/nunjucks/src/environment.js:527:18), <anonymous>:1045:3)
[11ty]     at Template.render (/Users/yndira/Documents/Yndira/flowforge/website/node_modules/nunjucks/src/environment.js:454:10)
[11ty]     at /Users/yndira/Documents/Yndira/flowforge/website/node_modules/@11ty/eleventy/src/Engines/Nunjucks.js:411:14
[11ty]     at new Promise (<anonymous>)
[11ty]     at /Users/yndira/Documents/Yndira/flowforge/website/node_modules/@11ty/eleventy/src/Engines/Nunjucks.js:410:14
[11ty]     at TemplateLayout.render (/Users/yndira/Documents/Yndira/flowforge/website/node_modules/@11ty/eleventy/src/TemplateLayout.js:236:31)
[11ty]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[11ty]     at async Template.renderPageEntry (/Users/yndira/Documents/Yndira/flowforge/website/node_modules/@11ty/eleventy/src/Template.js:793:17)
[11ty] Benchmark    538ms  11%   506× (Configuration) "jsmin" Nunjucks Async Filter
[11ty] Wrote 0 files (skipped 4) in 5.01 seconds (v2.0.1)
```
Because there's no URL, the permalink is set to false for those